spec/03-plugins/method-restriction/02-access_spec.lua

```diff
@@ -0,0 +1,116 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson"
+
+describe("Plugin: method-restriction (access)", function()
+  local client
+  setup(function()
+    helpers.kill_all()
```

Comment:
This shouldn't be necessary anymore. Tests are responsible for terminating pervious Kong instances in their `teardown` phase, per convention.